packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -75,6 +91,12 @@ EOF
 %{gem_instdir}/settings.d
 %{foreman_proxy_bundlerd_dir}/smart_proxy_ansible.rb
 %config %{foreman_proxy_settingsd_dir}/ansible.yml
+%attr(-,foreman-proxy,foreman-proxy) %{foreman_proxy_dir}/.ansible
+%attr(-,foreman-proxy,foreman-proxy) %{foreman_proxy_dir}/.ansible_galaxy
+%attr(-,foreman-proxy,foreman-proxy) %{foreman_proxy_dir}/.ansible.cfg
+%attr(-,foreman-proxy,foreman-proxy) %{_localstatedir}/lib/foreman_proxy/ansible
+%attr(-,foreman-proxy,foreman-proxy) %{_localstatedir}/lib/foreman_proxy/ansible_galaxy
+%attr(-,foreman-proxy,foreman-proxy) /etc/foreman-proxy/ansible.cfg
```

Comment:
For etc we have ``````%{_sysconfdir}```. I also think this should be owned by root. That means you can leave it out. It should be using ```%config``` though.

Reply:
How would `foreman-proxy` be able to read it if it is owned by root?

Reply:
By default it's 644. If there's a password in there ```%attr(0640,root,foreman-proxy)``` might be a better solution but you're right ```root:foreman-proxy``` is better in any case.